Sumemos/sumemos.js

```diff
@@ -0,0 +1,33 @@
+function isanumber (orden_num) {
```

Comment:
Utiliza lower camel case para el nombrado de funciones, variables, parámetros de funciones. Como buena práctica no utilices guión bajo para el nombre de variables, funciones, parámetros de funciones. Las constantes si pueden llevar el guión bajo por ejemplo:

> **_const BACKGROUND_COLOR = "red";_**

El prefijo **_is_** suele utilizarse para nombrar funciones que retornan un valor booleano como es el caso de la función 
is NaN. Para este caso el nombre de la función debería nombrarse de otro modo por ejemplo: getUserInput.

Reply:
Ok! cambiado.